docs/rendering_system.rst

```diff
@@ -0,0 +1,241 @@
+..
+   Author: Adam Turner
+
+   We can't use :pep:`X` references in this document, as they are relative from
+   the content root.
+   We use :doc:`PEP X <../pep-xxxx>` instead.
+
+
+An Overview of the PEP Rendering System
+=======================================
+
+This document provides an overview of the PEP rendering system, as a companion
+to :doc:`PEP 676 <../pep-0676>`.
+
+
+1. Configuration
+----------------
+
+Configuration is held in three files:
```

Comment:
I would say "held" as in "the water was held in the bucket". I'll change it to "stored" though.

Reply:
Yeah, we'd probably say "the water was held in the bucket" rather than "the water was stored in the bucket", but the meaning is slightly distinct ("held" implies the storage was more temporary and in advance of some other disposition, while "stored" implies long-term storage with no expectation of another immediate use"). To me it just didn't sound idiomatic to refer to "configuration" as being "held" and is not a usage I've ever really heard much.